api/src/constants/notifications.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Generic notfication messages
+ *
+ * @export
+ * @enum {number}
+ */
+export enum ACCESS_REQUEST_MESSAGE {
```

Comment:
An enum isn't quite the right structure for this, since the individual values aren't really stand alone items.

I would just define this is a const, and then the entire object becomes the message.

pseudo code:

```typescript
export const ACCESS_REQUEST_ADMIN_EMAIL: IgcNotifyGenericMessage = {
  header: 'awdawd',
  body1: 'awdawd',
  ...
}

// and then using it in the endpoint just becomes
gcnotifyService.sendEmailGCNotification(ADMIN_EMAIL, ACCESS_REQUEST_ADMIN_EMAIL);

```